src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionSettings.cs

```diff
@@ -48,6 +52,8 @@ internal sealed class HttpConnectionSettings
 
         internal IDictionary<string, object> _properties;
 
+        internal Func<string, int, CancellationToken, ValueTask<Stream>> _customConnect = ConnectHelper.ConnectAsync;
```

Comment:
This is going to allocate a new delegate for every HttpConnectionSettings.  While that's unlikely to be particularly meaningful, it's also unnecessary given that ConnectHelper.ConnectAsync is static.

Reply:
Refactoring oversight; thanks.